rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -336,19 +347,24 @@ def test_upgrade_v4_v5(globaldb):
             global_dir=globaldb._data_directory / 'global_data',
             db_filename=GLOBAL_DB_FILENAME,
         )
-
     assert globaldb.get_setting_value('version', None) == 5
-
+    # insert some values in unique_cache table.
+    with globaldb.conn.write_ctx() as write_cursor:
+        # check that values are replaced and not appended
+        globaldb_set_cache_values(
+            write_cursor=write_cursor,
+            key_parts=['TEST_KEY', '123'],
+            values=['klm'],
+        )
```

Comment:
This is not the right place to tests this. Here you are testing that the path with the special set of keys works for the function but this should be a unit test. What you should do is set

```
UNIQUE_CACHE_KEYS: set[GeneralCacheType] = {GeneralCacheType.CURVE_POOL_ADDRESS,
    GeneralCacheType.MAKERDAO_VAULT_ILK,GeneralCacheType.CURVE_GAUGE_ADDRESS}
```

and then verify that the tests for this keys still work and are stored in the correct table. This is not the right place to test this